tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
We might want to add:

`"isolatedModules": true`
ref. https://www.typescriptlang.org/tsconfig#isolatedModules